README.md

```diff
@@ -30,3 +30,10 @@ Licensing and Pricing options are customized to your usage and requirements. For
 - Printing
 - Security - Password, encrypt, watermark
 - Search for and replace or edit text
+
+## Instructions for trying out the samples
+- Clone this repository to your system (64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon)
+- Select a sample you would like to try
+- Open the folder containing the sample in [IntelliJ IDEA](https://www.jetbrains.com/idea/)
```

Comment:
I think we need a step to remind users to press the Load Maven Project if this box appears.
<img width="330" alt="image" src="https://github.com/datalogics/apdfl-java-maven-samples/assets/89971/cabe0111-e421-4743-90a8-a5a46de84776">

Reply:
This seems to be unique to your system, with only the one sample, I'll fix it when we have more info